tests/Support/EmitterTest.php

```diff
@@ -102,17 +110,35 @@ public function testBindPriority()
         $emitter = $this->traitObject;
         $result = '';
 
-        $emitter->bindEvent('event.test', function() use (&$result) { $result .= 'the '; }, 90);
-        $emitter->bindEvent('event.test', function() use (&$result) { $result .= 'quick '; }, 80);
-        $emitter->bindEvent('event.test', function() use (&$result) { $result .= 'brown '; }, 70);
-        $emitter->bindEvent('event.test', function() use (&$result) { $result .= 'fox '; }, 60);
-        $emitter->bindEvent('event.test', function() use (&$result) { $result .= 'jumped '; }, 50);
-        $emitter->bindEvent('event.test', function() use (&$result) { $result .= 'over '; }, 40);
-        $emitter->bindEvent('event.test', function() use (&$result) { $result .= 'the '; }, 30);
-        $emitter->bindEvent('event.test', function() use (&$result) { $result .= 'lazy '; }, 20);
-        $emitter->bindEvent('event.test', function() use (&$result) { $result .= 'dog'; }, 10);
+        $emitter->bindEvent('event.test', function () use (&$result) {
+            $result .= 'the ';
+        }, 90);
+        $emitter->bindEvent('event.test', function () use (&$result) {
+            $result .= 'quick ';
+        }, 80);
+        $emitter->bindEvent('event.test', function () use (&$result) {
+            $result .= 'brown ';
+        }, 70);
+        $emitter->bindEvent('event.test', function () use (&$result) {
+            $result .= 'fox ';
+        }, 60);
+        $emitter->bindEvent('event.test', function () use (&$result) {
+            $result .= 'jumped ';
+        }, 50);
+        $emitter->bindEvent('event.test', function () use (&$result) {
+            $result .= 'over ';
+        }, 40);
+        $emitter->bindEvent('event.test', function () use (&$result) {
+            $result .= 'the ';
+        }, 30);
+        $emitter->bindEvent('event.test', function () use (&$result) {
+            $result .= 'lazy ';
+        }, 20);
+        $emitter->bindEvent('event.test', function () use (&$result) {
+            $result .= 'dog';
+        }, 10);
```

Comment:
hmm, the earlier version was definitely easier to read, is there anyway we can allow for this?